packages/multicore-magic/multicore-magic.1.0.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Low-level multicore utilities for OCaml"
+maintainer: "Vesa Karvonen <vesa.a.j.k@gmail.com>"
+authors: "Vesa Karvonen"
+license: "0BSD"
+homepage: "https://github.com/ocaml-multicore/multicore-magic"
+bug-reports: "https://github.com/ocaml-multicore/multicore-magic/issues"
+dev-repo: "git+https://github.com/ocaml-multicore/multicore-magic.git"
+depends: [
+  "dune" {>= "3.3.0"}
+  "ocaml" {>= "5.0.0"}
+  "ocamlformat" {= "0.24.1" & with-test}
```

Comment:
This does not seem to be used in the tests and tests are not enabled in this opam file anyway
```suggestion
#  "ocamlformat" {= "0.24.1" & with-test}
```